pkg/services/ngalert/store/alertmanager.go

```diff
@@ -154,7 +154,7 @@ func (st *DBstore) MarkConfigurationAsApplied(ctx context.Context, cmd *models.M
 		}
 
 		if rowsAffected != 1 {
-			st.Logger.Error("update statement affected %d alert configuration history records", rowsAffected)
+			st.Logger.Warn("Alert configuration history updated", "rows", rowsAffected)
```

Comment:
```suggestion
			st.Logger.Warn("No alertmanager configs were found to mark as applied", "org", cmd.OrgID, "hash", cmd.ConfigurationHash)
```
Due to the Limit(1) it's only possible to hit this when `rowsAffected` is `0`, so the rows attribute isn't very useful. Including the hash can help when debugging.

Reply:
Or that all configs are marked, that's why I didn't want to specify a certain scenario.

Reply:
@grobinson-grafana's wording above is also good, I opened my browser prior to that commit so I didn't see it